vite.config.ts

```diff
@@ -3,6 +3,12 @@ import { defineConfig } from 'vite';
 import tsconfigPaths from 'vite-tsconfig-paths';
 import { netlifyPlugin } from '@netlify/remix-adapter/plugin';
 
+declare module '@remix-run/node' {
+    interface Future {
+        v3_singleFetch: true;
```

Comment:
this augments the type and ensures `useLoaderData` can infer the correct type.
it is the default in current `create-remix` generated apps